tensorboard/plugins/hparams/tf_hparams_backend/tf-hparams-backend.html

```diff
@@ -48,6 +48,17 @@
           return this._sendRequest('experiment', experimentRequest);
         }
 
+        getDowloadURL(format, listSessionGroupsRequest) {
```

Comment:
sp.: *dow**n**load*, and [please capitalize as `Url`][1].

[1]: https://google.github.io/styleguide/javaguide.html#s5.3-camel-case